frontend/packages/console-plugin-sdk/src/typings/pages.ts

```diff
@@ -1,6 +1,6 @@
-import { RouteProps, RouteComponentProps } from 'react-router-dom';
-import { K8sKind, K8sResourceKindReference } from '@console/internal/module/k8s';
 import { Extension, LazyLoader } from './base';
+import { K8sKind, K8sResourceKindReference } from '@console/internal/module/k8s';
+import { RouteComponentProps, RouteProps } from 'react-router-dom';
```

Comment:
Nit: vendor (3rd party) imports should be listed before Console pkg imports.

(But it seems that current ESLint config doesn't enforce that..)